Compilation error with MongoDB 3.5.2

New MongoDB cause build break with PMSE. 

Commit ID: 1346352ff66b163fb010a270d1826c18699f3b36

Log:
`Compiling build/opt/mongo/db/commands/index_filter_commands_test.o
Compiling build/opt/mongo/db/modules/pmse/pmse_init.o
In file included from src/mongo/db/modules/pmse/pmse_init.cpp:33:0:
src/mongo/db/modules/pmse/pmse_engine.h:43:43: fatal error: mongo/db/storage/kv/kv_engine.h: No such file or directory
compilation terminated.
Linking build/opt/mongo/db/commands/list_collections_filter_test
Generating placeholder library build/opt/mongo/db/query/libquery_test_service_context.a
Skipping ranlib for build/opt/mongo/db/query/libquery_test_service_context.a
Compiling build/opt/mongo/db/modules/pmse/pmse_engine.o
scons: *** [build/opt/mongo/db/modules/pmse/pmse_init.o] Error 1
Compiling build/opt/mongo/db/modules/pmse/pmse_record_store.o
src/mongo/db/modules/pmse/pmse_engine.cpp:36:34: fatal error: mongo/platform/basic.h: No such file or directory
compilation terminated.
scons: *** [build/opt/mongo/db/modules/pmse/pmse_engine.o] Error 1
src/mongo/db/modules/pmse/pmse_record_store.cpp:35:43: fatal error: mongo/db/storage/record_store.h: No such file or directory
compilation terminated.
scons: *** [build/opt/mongo/db/modules/pmse/pmse_record_store.o] Error 1
scons: building terminated because of errors.
build/opt/mongo/db/modules/pmse/pmse_init.o failed: Error 1
build/opt/mongo/db/modules/pmse/pmse_engine.o failed: Error 1
build/opt/mongo/db/modules/pmse/pmse_record_store.o failed: Error 1
`

--- a/SConscript
+++ b/SConscript
@@ -1,4 +1,9 @@
+# -*- mode: python -*-
 Import("env")
+
+env = env.Clone()
+
+env.InjectMongoIncludePaths()
 
 env.Library(
     target= 'storage_pmse_base',

